node_modules/oae-principals/lib/rest.user.js

```diff
@@ -784,3 +784,30 @@ OAE.tenantRouter.on('get', '/api/user/:userId/groups/recent', function(req, res)
         return res.send(200, {'results': results});
     });
 });
+
+/**
+ * @REST getUsersForTenant
+ *
+ * Get all users for a given tenant
+ *
+ * @Server      admin,tenant
+ * @Method      GET
+ * @Path        /tenant/{tenantAlias}/users
```

Comment:
Let's make it `tenants` as plurals are the [best and common practice](http://www.restapitutorial.com/lessons/restfulresourcenaming.html).